src/commands/deployImage/imageSource/buildImageInAzure/BuildImageStep.ts

```diff
@@ -40,10 +42,36 @@ export class BuildImageStep extends AzureWizardExecuteStep<IBuildImageInAzureCon
             context.errorHandling.suppressDisplay = true;
             throw new Error(errorMessage);
         }
+
+        // Need to place this outside of 'initSuccessOutput' so we can use the image after it has had a chance to become defined
+        (this.success.output as string[])?.push(localize('useImage', 'Using image "{0}".', context.image));
```

Comment:
I'm not entirely sure why this is part of the `successOutput`. Wouldn't you want this to be displayed even if it fails? I'm also not sure what's the difference between `context.image` and `context.imageName`?

Reply:
`context.image` is a more full version of the image name.  It would include the full azurecr.io/etc...

We wouldn't want to show that as part of the fail output because if this fails, the whole command fails and we wouldn't be able to proceed any further to use the image (I'm assuming the image would fail to build)